lib/theme/textStyles.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
```

Comment:
https://dart.dev/guides/language/effective-dart/style#do-name-libraries-and-source-files-using-lowercase_with_underscores